Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -47,6 +47,28 @@ public static void UseOrganisationEndpoints(this WebApplication app)
                 return operation;
             });
 
+        app.MapPost("/admin/organisation/review", async (ReviewOrganisation command, IUseCase<ReviewOrganisation, Boolean> useCase) =>
```

Comment:
@jakzal What do you think this endpoint should be? Is it ok how it is or would you prefer something else?

Reply:
Imo we should follow the same pattern as everywhere else. We're working with an organisation resource, so it should be under `/organisations/{organisationId}/...`.

Using `PATCH /organisations/{organisationId}` for any organisation updates would be consistent with the approach the team has used in the past.

Reply:
So in this case

PATCH /organisations/{organisationId}/review

Since its not just a simple CRUD task